examples/session_controller.rb

```diff
@@ -1 +1,16 @@
-# TODO
+class SessionController < ApplicationController
+  skip_user_authentication only: [:new]
+
+  def new
+  end
+
+  def create
+    if user = Autho::Authentication.new(User, params[:email], params[:password]).user
+      Autho::UserSession.new(self, :user_id, User).user = user
```

Comment:
This is just `Autho::Session`, not `UserSession`.

In the project that uses this we do `user_session.user = user` since it reads a bit better and we can reuse it elsewhere, e.g. logout (`user_session.clear`).